index.js

```diff
@@ -9,24 +8,23 @@ const ENV_INFO = `${testCafePkg.name}/${testCafePkg.version}`;
 
 // Take a DOM snapshot and post it to the snapshot endpoint
 module.exports = async function percySnapshot(t, name, options) {
-  // if name is the first arg, assume an implicit controller
-  if (!t || typeof t === 'string') [t, name, options] = [implicit, t, name];
+  if (!t) throw new Error('The `test` argument is required.');
```

Comment:
Should we use the same verbiage as TestCafe does?

```suggestion
  if (!t) throw new Error('The test controller argument is required.');
```

Reply:
Or 
```suggestion
  if (!t) throw new Error('The test function\'s `t` argument is required.');
```